docs/dynamic-plugins/installing-external-plugins.md

```diff
@@ -0,0 +1,181 @@
+
+To distribute the derived dynamic plugin package as a JavaScript package, publish it to a private or internal npm registry. Avoid using the public npm registry (npmjs.com), as the derived dynamic plugin package is structured like regular JavaScript packages but cannot be used the same way.
+
+```bash
+cd dist-dynamic
+npm publish --registry <npm_registry_url>
+```
+
+Alternatively, add the following to your `package.json` before running `npx @janus-idp/cli@latest package export-dynamic-plugin`:
+
+```json
+{
+  "publishConfig": {
+    "registry": "<npm_registry_url>"
+  }
+}
+```
+
+If you modify the `publishConfig` after running `export-dynamic-plugin`, run it again to ensure the `dist-dynamic` directory contains a `package.json` with the correct `publishConfig`.
+
+## Installing External Dynamic Plugins
+
+Dynamic plugins can be packaged in three formats:
+
+- OCI image
+- `tgz` archive
+- npm package
+
+You can also load the dynamic plugin from a plain directory, though this is not recommended for production use. It can be helpful for development and testing.
+
+To install a dynamic plugin, you need to add the plugin to the `dynamic-plugin-config.yaml` file.
+
+The placement of `dynamic-plugin-config.yaml` depends on the deployment method. For more information, see [Installing Dynamic Plugins with the Red Hat Developer Hub Operator](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/installing_and_viewing_dynamic_plugins/proc-config-dynamic-plugins-rhdh-operator_title-plugins-rhdh-about) or [Installing Dynamic Plugins Using the Helm Chart](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/installing_and_viewing_dynamic_plugins/con-install-dynamic-plugin-helm_title-plugins-rhdh-about).
+
+Plugins are defined in the `plugins` array in the `dynamic-plugin-config.yaml` file. Each plugin is defined as an object with the following properties:
+
+- `package`: The package definition of the plugin. This can be an OCI image, `tgz` archive, npm package, or a directory path.
+- `disabled`: A boolean value that determines whether the plugin is enabled or disabled.
+- `integrity`: The integrity hash of the package. This is required for `tgz` archives and npm packages.
+- `pluginConfig`: The configuration for the plugin. This is optional and can be used to pass configuration to the plugin. Anything that is added to this object will be merged with the main app-config.
+
+### Loading a Plugin from an OCI Image
+
+When defining the plugin packaged as an OCI image, use the `oci://` prefix, followed by the image name, tag, and plugin name separated by the `!` character (`oci://<image-name>:<tag>!<plugin-name>`).
+
+```yaml
+plugins:
+  - disabled: false
+    package: oci://quay.io/example/image:v0.0.1!backstage-plugin-myplugin
+```
+
+### Loading a Plugin from a `tgz` Archive
```

Comment:
The term "loading" starts to feel overloaded to me at this point.  These section headers are really for sections that are discussing how to refer to these packages from the configuration.  Maybe "Configuring a .tgz archive URL" or something like that for this and the other ones?

Reply:
I would like to keep heading consistend. What about "Using tgz archive"? To match what you suggested for next section heading?

Reply:
Yup, "Using a tgz Archive".  Also "Referencing a tgz archive", "Referencing an OCI Container" etc could be an option here maybe too.